src/R/berekening_hulpvariabelen.R

```diff
@@ -189,6 +222,21 @@ read_sbp_akkervogels <- function(
       mutate(Naam = gebied$Naam) %>%
       rename(geometry = x)
   }
+
+  if (!is.null(path_extra_soorten)) {
+    if (is.null(extra_soorten)) {
+      stop(paste0("Specificeer welke extra soorten moeten worden toegevoegd! ",
+                  "Argument 'extra_soorten'."), call. = FALSE)
+    }
+
+    sbp_others <- read_sbp_others(path_extra_soorten, extra_soorten, gebied)
+    out <- bind_rows(sbp_akkervogels_final, sbp_others) %>%
+      group_by(Naam) %>%
+      summarise()
```

Comment:
I do this to keep the `Naam` column in 'out'. I feel this is more elegant than: `st_union() %>% mutate(Naam = gebied$Naam)`

Or is there a reason why you would do this?

Reply:
I see. You can keep it. Since Naam is "gebied$Naam", the result will be one polygon containing the union of SBP akkervogels and all other relevant related SBP's. I was under the false impression that Naam referred to the names of the different GIS layers involved.